c7n/utils.py

```diff
@@ -343,3 +345,27 @@ def _f(*args, **kw):
             raise
     functools.update_wrapper(_f, f)
     return _f
+
+
+def int_from_value(raw_val, default=0):
+    if isinstance(raw_val, int):
+        return raw_val
+
+    try:
+        value = int(raw_val)
+    except ValueError:
+        value = default
+    return value
+
+
+def datetime_from_value(raw_val, default=None):
+    if isinstance(raw_val, datetime):
+        return raw_val
+
+    if default is None:
+        default = datetime.now(tz=tzutc())
```

Comment:
If you don’t pass in an alternate default, that’s true, but if you look at the places where this utility is called, you’ll see that the default should be overridden to match what was there previously. (There are tests that would have broken otherwise.)